src/pks/flow/constitutive_relations/wrm/rel_perm_frzBC_evaluator_reg.hh

```diff
@@ -0,0 +1,20 @@
+/*
+  Copyright 2010-202x held jointly by participating institutions.
+  ATS is released under the three-clause BSD License.
+  The terms of use and "as is" disclaimer for this license are
+  provided in the top-level COPYRIGHT file.
+
+  Authors:
+*/
+
+#include "rel_perm_frzBC_evaluator.hh"
+
+namespace Amanzi {
+namespace Flow {
+
+// registry of method
+Utils::RegisteredFactory<Evaluator, RelPermFrzBCEvaluator>
+  RelPermFrzBCEvaluator::factory_("FrzBC rel perm");
```

Comment:
Please come up with a better, more explicit name.  "FrzBC" doesn't mean anything to me.  Longer is better in this case!

Reply:
You can leave the file named what it is, just update the "evaluator type"!